gulp_tasks/sass.js

```diff
@@ -12,5 +13,10 @@ gulp.task('sass', function () {
         browsers: config.sass.autoprefixer.browsers
       })
     ]))
+    .pipe(
```

Comment:
I'm not entirely sure how gulp-rename works, but added this here.

Reply:
Yes, that's correct.